### ISSUE TITLE
using `-test`

### ISSUE THREAD

**Issue by hroest** [OpenMS/OpenMS#2325]
especially after the update check we should ensure that all our tests are run with `-test`

currently it does not look so good:

```
$ grep add_test src/tests/topp/CMakeLists.txt  | grep  '\-test' | wc
    392    4223   89888
$ grep add_test src/tests/topp/CMakeLists.txt  | grep -v  '\-test' | wc
    463    3757   70810
```

**Comment by timosachsenberg** [comment]
hmm actually they should be now. When I run all tests I don't get any REST requests or .ver files in the .OpenMS folder

**Comment by hroest** [comment]
strange, as it seems many `add_test` statements do not have the `-test` flag

**Comment by timosachsenberg** [comment]
maybe fuzzydiff ones?

**Comment by hroest** [comment]
true, I guess fuzzydiff does not call the usage page?

```
$ grep add_test src/tests/topp/CMakeLists.txt  | grep -v  '\-test' | grep -v DIFF | wc
     18      67    1499
$ grep add_test src/tests/topp/CMakeLists.txt  | grep -v  '\-test' | grep -v DIFF | less -S
# Test names (the first argument of 'add_test') must be distinct.
add_test("TOPP_WSDL_1" ${TOPP_BIN_PATH}/AdditiveSeries -write_wsdl WSDL_out.tmp)
add_test("TOPPWRITEINI_GenericWrapper" ${TOPP_BIN_PATH}/GenericWrapper -type RAWFileConvert -write_ini "GenericWrapper.ini.tmp")
  add_test("TOPPWRITEINI_${i}" "${TOPP_BIN_PATH}/${i}" -write_ini "${i}.ini.tmp")
        add_test("TOPPWRITEINI_${i}_SectionName" ${CMAKE_COMMAND} -DNAME=${i} -DFILE=${i}.ini.tmp -P "${DATA_DIR_TOPP}/check_ini.cmake")
  add_test("UTILSWRITEINI_${i}" "${TOPP_BIN_PATH}/${i}" -write_ini "${i}.ini.tmp")
        add_test("UTILSWRITEINI_${i}_SectionName" ${CMAKE_COMMAND} -DNAME=${i} -DFILE=${i}.ini.tmp -P "${DATA_DIR_TOPP}/check_ini.cmake")
  add_test("UTILSWRITECTD_${i}" "${TOPP_BIN_PATH}/${i}" -write_ctd)
  add_test("TOPPWRITECTD_${i}" "${TOPP_BIN_PATH}/${i}" -write_ctd)
add_test("TOPPWRITEINI_OVERWRITE" ${TOPP_BIN_PATH}/PeakPickerWavelet -write_ini "WRITE_INI.ini.tmp" -ini "${DATA_DIR_TOPP}/WRITE_INI_IN.ini")
add_test("TOPP_HighResPrecursorMassCorrector_1"
add_test("TOPP_HighResPrecursorMassCorrector_2"
  add_test("TOPP_OpenSwathAssayGenerator_test_1"
  add_test("TOPP_OpenSwathDecoyGenerator_test_1"
  add_test("TOPP_OpenSwathDecoyGenerator_test_2"
  add_test("TOPP_OpenSwathDecoyGenerator_test_3"
  add_test("TOPP_OpenSwathFileSplitter_1" ${TOPP_BIN_PATH}/OpenSwathFileSplitter -in ${DATA_DIR_TOPP}/OpenSwathWorkflow_1_input.mzML -outputDirectory "")
add_test("UTILS_SimpleSearchEngine_1" ${TOPP_BIN_PATH}/SimpleSearchEngine
```

so most of these are fine

**Comment by hroest** [comment]
see also #2326

**Comment by hroest** [comment]
closed through #2326
